src/dynamic_graph_head/force_plate.py

```diff
@@ -0,0 +1,64 @@
+"""
+License: BSD 3-Clause License
+Copyright (C) 2021, New York University
+
+Copyright note valid unless otherwise stated in individual files.
+All rights reserved.
+"""
+#NOTE: This is only a simulation class for now, we don't have an actual force plate 
+
+from matplotlib.pyplot import hist
+import numpy as np
+import pinocchio as pin
+
+
+class SimForcePlate:
+    def __init__(self, objects, delay_dt=0, noise_data_std={}):
```

Comment:
As `noise_data_std` is not implemented yet, maybe it's best to not provide it as an argument? This way, if someone provides the data they don't get confused why it's not getting applied.